docs/howtoguides/how_to_roll_out_across_multiple_machines.rst

```diff
@@ -0,0 +1,48 @@
+.. _roll_out_multiple_machines:
+
+How to roll out your pro token across multiple machines
+=======================================================
+
+This guide will show you how to roll out your pro token across multiple machines using cloud-init for configuration.
+
+Configure using Cloud-init
```

Comment:
Can probably frame this better. Any suggestions?

Reply:
I'll probably have more suggestions once all the content is in place - let's leave this as a working title for now